events.html

```diff
@@ -71,583 +71,585 @@ <h5 class="header col s12 light">
-          </span>
-          10:00 - 16:00
-        </h5>
-        <div class="event">
-
-          <div class="title">
-            <a target="_blank" class="grey-text" href="https://nodegirls.typeform.com/to/wkVUzc">
-              Introduction to Node.js and JavaScript
-            </a>
-            at
-            <a target="_blank" class="grey-text" href="https://pusher.com">
-              Pusher
-            </a>
-          </div>
-
-          <p class="flow-text">
-            This is a combo workshop aimed at both absolute beginners and folk with some programming experience.
-            <br/>Learn JavaScript and programming basics.  Or, learn to build your first server with Node.js.
-          </p>
+        <div class="event-container">
+          <a href="london.html"><div class="chip">London</div></a>
+          <h5 class="header light event-container__header">Saturday 30<sup>th</sup> September 2017</h5>
+          <h5 class="header time">
+            <span class="inline-clock-icon">
+              <svg width="11" height="11" viewBox="0 0 11 11"><path d="M5.4 0C2.4 0 0 2.4 0 5.4s2.4 5.4 5.4 5.4 5.4-2.4 5.4-5.4S8.4 0 5.4 0zm3 6.8H4.7V1.7h.7L6 5.4l2.4.6v.8z"></path></svg>
+            </span>
+            10:00 - 16:00
+          </h5>
+          <div class="event">
 
-          <div class="row">
-            <div class="organisers-container col m6">
-              <a target="_blank" href="https://twitter.com/claireinez">
-                <div class="team-image team-image__small claire" src="" alt="Claire Mitchell"></div>
-              </a>
-              <a target="_blank" href="https://www.twitter.com/msmichellegar">
-                <div class="team-image team-image__small michelle" src="" alt="Michelle Garrett"></div>
+            <div class="title">
+              <a target="_blank" class="grey-text" href="https://nodegirls.typeform.com/to/FC57Oj">
+                Introduction to Node.js and JavaScript
               </a>
-              <a target="_blank" href="https://twitter.com/minaorangina">
-                <div class="team-image team-image__small mina" src="" alt="Mina Gyimah"></div>
+              at
+              <a target="_blank" class="grey-text" href="http://www.ticketmaster.com">
+                Ticketmaster
               </a>
             </div>
 
-            <div class="col m6 sponsors-container">
-              <a target="_blank" href="https://pusher.com">
-                <img class="sponsor" src="images/sponsors/pusher-logo.png" alt="pusher-logo">
-              </a>
+            <p class="flow-text">
+              This is a combo workshop aimed at both absolute beginners and folk with some programming experience.
+              <br/>Learn JavaScript and programming basics.  Or, learn to build your first server with Node.js.
+              <br/>Applications have now closed.
```

Comment:
This line needs to be removed so might as well do it now :)

Reply:
Good catch :)